gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -246,6 +256,19 @@ protected void processMessage(DecodeableKafkaRecord<byte[],byte[]> message) {
     }
   }
 
+  private void clearStartDeadlineTriggerAndDagAction(String flowGroup, String flowName, String flowExecutionId, String jobName) {
+
+    DagActionStore.DagAction enforceStartDeadlineDagAction = new DagActionStore.DagAction(flowGroup, flowName,
+        String.valueOf(flowExecutionId), jobName, DagActionStore.DagActionType.ENFORCE_START_DEADLINE);
+    try {
+      GobblinServiceManager.getClass(DagActionReminderScheduler.class).unscheduleReminderJob(enforceStartDeadlineDagAction);
+      GobblinServiceManager.getClass(DagActionStore.class).deleteDagAction(enforceStartDeadlineDagAction);
+    } catch (SchedulerException | IOException e) {
+      log.warn("Failed to unschedule the reminder for {}", enforceStartDeadlineDagAction);
```

Comment:
error level logging maybe. although its possible the scheduler reminder fired and was deleted so that can be warn while the other is error.